tests/wpunit/CommentObjectCursorTest.php

```diff
@@ -3,6 +3,9 @@
 
 class CommentObjectCursorTest extends \Tests\WPGraphQL\TestCase\WPGraphQLTestCase {
     public function testThresholdFieldsQueryVar() {
+        // Skip until bugs in CommentObjectCursor class are resolved.
```

Comment:
What bugs? Do they have existing tickets?

Reply:
It's an ordering bug and no I haven't written a ticket on it yet.